### PR TITLE
Fix the `CodeGenerationFailed` notification code

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-166.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-166.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -633,10 +633,10 @@ a|The database was unable to generate code for the query. A stacktrace can be fo
 |Category
 m|PERFORMANCE
 |GQLSTATUS code
-m|01N40
+m|03N96
 |Status description
-a|warn: unsupported runtime.
-The query cannot be executed with `preparser_input1`, `preparser_input2` is used. Cause: `$msg`.
+a|info: failed code generation.
+The database was unable to generate code for the query. A stacktrace can be found in the debug.log. Cause: $msg.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:639]:
```suggestion
Failed to generate code, falling back to interpreted $enginetype engine. A stacktrace can be found in the debug.log. Cause: $msg.
```

**Review by renetapopova — COMMENTED** [review]
I left a comment

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-166.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-166.surge.sh/status-codes/5/notifications/all-notifications)

**Review by Lojjs — APPROVED** [review]

**Pull request merged**